week05/errors_test.go

```diff
@@ -0,0 +1,190 @@
+package demo
+
+import (
+	"testing"
+)
+
+func Test_errors_Error(t *testing.T) {
```

Comment:
Try to use clearer and more descriptive test names.

```suggestion
func Test_IsErrTableNotFound(t *testing.T) {
```